tls/s2n_kem.c

```diff
@@ -74,23 +162,56 @@ const struct s2n_kem_group s2n_secp256r1_kyber_512_r3 = {
     .kem = &s2n_kyber_512_r3,
 };
 
-#if EVP_APIS_SUPPORTED
+const struct s2n_kem_group s2n_secp256r1_kyber_768_r3 = {
+    .name = "secp768r1_kyber-768-r3",
```

Comment:
This line has a typo, it starts with `secp768r1` instead of `secp256r1`. But I also had another question, did we want to change the name to `SecP256r1Kyber768Draft00` or similar? We're using the ID from [this draft RFC, did we also want to use their name too](https://datatracker.ietf.org/doc/html/draft-kwiatkowski-tls-ecdhe-kyber-01#section-5-2.2.1)?

Reply:
thanks for catching that, and good question. I think the answer might depend on what this field is used for. Only the IANA identifier should be sent over the wire, so presumably this is just for display and debugging purposes? The [linked section of RFC 8447](https://datatracker.ietf.org/doc/html/draft-ietf-tls-rfc8447bis-04#section-6) doesn't appear to stipulate the "Description" field's purpose.

Reply:
Yes, it's used for display and debugging purposes, but also used in TLS dashboards, so changing the display name at a later time will break dashboard history if it's changed in the future.

Reply:
Got it. I've updated the name fields to use the descriptions specified by IANA.